docs/whats-new.md

```
@@ -13,19 +13,20 @@ Stay up to date with every PyTrendy release — user-facing improvements, bug fi
 ```bash
 pip install --pre pytrendy
 ```
 
-??? note "Zero-baseline market entry Up detection"
-    One fix in v1.2.0-dev.6: corrects an off-by-one in segment extraction that suppressed short Up trends on zero-baseline market entries, and preserves Down `total_change` metadata.
+??? note "Zero-baseline market entry — detection improvements"
+    Multiple fixes since v1.2.0 have improved trend detection on zero-baseline and new-market entry series. These fixes address edge cases where the algorithm incorrectly suppressed trends or introduced spurious noise segments on series that start at zero.
 
-    Short-lived Up trends (≥3 days) emerging from a long zero baseline were lost due to an off-by-one error in `get_segments()` that failed to count the first point of a new direction segment. This primarily affected smaller ramps (e.g. 10→125 over 5 days) on new-market or quasi-experimental series.
-    [#171](https://github.com/RussellSB/pytrendy/issues/171) [#177](https://github.com/RussellSB/pytrendy/issues/177)
+    #### Up trend detection on smaller ramps
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)